js/src/components/google-combo-account-card/connect-ads/connect-ads.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AccountCard from '.~/components/account-card';
+import useApiFetchCallback from '.~/hooks/useApiFetchCallback';
+import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import { useAppDispatch } from '.~/data';
+import useExistingGoogleAdsAccounts from '.~/hooks/useExistingGoogleAdsAccounts';
+import ConnectAccountCard from '../connect-account-card';
+import ConnectAdsFooter from './connect-ads-footer';
+import ConnectAdsBody from './connect-ads-body';
+
+/**
+ * ConnectAds component renders an account card to connect to an existing Google Ads account.
+ *
+ * @fires gla_documentation_link_click with `{ context: 'setup-ads-connect-account', link_id: 'connect-sub-account', href: 'https://support.google.com/google-ads/answer/6139186' }`
+ * @return {JSX.Element} {@link AccountCard} filled with content.
+ */
+const ConnectAds = () => {
```

Comment:
Added the condition in component so that it will only be visible when account is not connected or when account is connected, but editing.

```js
if ( isConnected && ! isEditing ) {
    return null;
}
```

Reply:
This makes sense to me and the isEditing prop can be incorporated in #2605.